cache-config/t3c-apply/config/config.go

```diff
@@ -188,6 +190,29 @@ func directoryExists(dir string) (bool, os.FileInfo) {
 	return info.IsDir(), info
 }
 
+const RpmDir = "/var/lib/rpm"
```

Comment:
Exported constants should have GoDoc comments - alternatively, it looks like this isn't used outside of the file in which it is declared, so it doesn't need to be exported at all.

Reply:
Add to the comment that it isn't exported?

Reply:
changed to `rpmDir` so it will not be exported.